pdc/apps/unreleasedvariant/admin.py

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from pdc.apps.common.register_to_admin import register
+
+register('unreleasedvariant')
```

Comment:
If there are no models in the app, then registering to admin does not make much sense to me. Can this file be removed?

Reply:
Sure